frontend/main/components/NavigationMenu/NavigationMenu.js

```diff
@@ -57,10 +57,8 @@ export class NavigationMenu extends BaseComponent {
       button.appendChild(textElement);
 
       // Add icon second
-      const icon = document.createElement("img");
-      icon.src = link.src;
-      icon.alt = `${link.text} icon`;
-      icon.classList.add("nav-icon");
+      const icon = document.createElement("i");
```

Comment:
Right, we may not need a icon.alt since we have the text in button. Thank you for cleaning the codes!